### PR TITLE
Remove WPILib.h and old header guards

### LINKED ISSUE #155: Remove references to frc/WPILib.h
There are some code snippets throughout the docs (copied from screensteps) that reference WPILib.h

These will need to be updated following WPILib.h deprecation, see https://github.com/wpilibsuite/allwpilib/pull/1779

### PR DESCRIPTION
**Pull request by prateekma**
Semi-solves #155

### CODE REVIEW

File: source/docs/software/wpilib-tools/robotbuilder/advanced/robotbuilder-drive-mecanum.rst [hunk 1]
```diff
@@ -17,7 +17,8 @@ Code for Driving with Mecanum Wheels
 
    .. code-tab:: cpp
 
-      #include "WPILib.h"
+      #include "frc/RobotDrive.h"
+      #include "frc/Joystick.h"
```
Comment (AustinShalit) [inline source/docs/software/wpilib-tools/robotbuilder/advanced/robotbuilder-drive-mecanum.rst]: Flip for alphabetical order

File: source/docs/software/wpilib-tools/robotbuilder/writing-code/robotbuilder-writing-pidsubsystem-code.rst [hunk 2]
```diff
@@ -82,8 +82,7 @@ Adding the Constants to the Header File (C++ Only)
 
    .. code-tab:: cpp
 
-      #ifndef ELEVATOR_H
-      #define ELEVATOR_H
+      #pragma once
 
       #include "Commands/PIDSubsystem.h"
       #include "WPILIb.h"
```
Comment (AustinShalit) [inline source/docs/software/wpilib-tools/robotbuilder/writing-code/robotbuilder-writing-pidsubsystem-code.rst]: 🤔

**Comment by AustinShalit** [comment]
There is one more on this page: https://github.com/wpilibsuite/frc-docs/blob/master/source/docs/software/old-commandbased/commands/limit-switches-control-behavior.rst

**Comment by prateekma** [comment]
I am out of town until 12/30. Will address requested changes after that.

**Comment by AustinShalit** [comment]
Going to merge but not close the issue. Open a new PR when you get back.

**Pull request merged**